app/services/dunning_campaigns/bulk_process_service.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module DunningCampaigns
+  class BulkProcessService < BaseService
+    def call
+      return result unless License.premium?
+
+      eligible_customers.find_each do |customer|
+        CustomerDunningEvaluator.call(customer)
+      end
+
+      result
+    end
+
+    private
+
+    def eligible_customers
+      Customer
+        .joins(:organization)
+        .where(exclude_from_dunning_campaign: false)
+        .where("organizations.premium_integrations @> ARRAY[?]::varchar[]", ['auto_dunning'])
```

Comment:
Has the name `auto_dunning` been validated by the product team?

Reply:
can't remember, is this name visible for users?

frontend is using it... I'll check with product.

Thanks